inference-engine/src/mkldnn_plugin/mkldnn_graph.cpp

```diff
@@ -1200,6 +1200,11 @@ void MKLDNNGraph::EnforceBF16() {
     if (implication(isQuantized(), config.manualEnforceBF16)) {
         for (auto &node : graphNodes) {
             if (node->getType() != Input && node->getType() != Output) {
+                //  Softmax terminal layer will be executed in FP32 precision to maintain accuracy
+                if (node->getType() == Softmax &&
+                    node->getChildEdgeAt(0)->getChild()->getType() == Output) {
+                    continue;
```

Comment:
What if we do not have an output node on port 0, but there is one on the first port? Should we check all edges?

Reply:
Done